rfcs/0028-ckb-vm-version-1/0028-ckb-vm-version-1.md

```diff
@@ -0,0 +1,147 @@
+---
+Number: "0028"
+Category: Standards Track
+Status: Proposal
+Author: Wanbiao Ye
+Organization: Nervos Foundation
+Created: 2021-05-25
+---
+
+# VM version1
```

Comment:
It's better to mention the RFC0003 and add the backlink in RFC0003 to this RFC so when developers read RFC0003, they known there's a new version of VM.